nav2_map_server/include/nav2_map_server/map_3d/map_server_3d.hpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2020 Shivam Pandey
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_MAP_SERVER__MAP_3D__MAP_SERVER_3D_HPP_
+#define NAV2_MAP_SERVER__MAP_3D__MAP_SERVER_3D_HPP_
+
+#include <string>
+#include <memory>
+
+#include "nav2_msgs/srv/get_map3_d.hpp"
+#include "nav2_msgs/srv/load_map3_d.hpp"
+#include "sensor_msgs/msg/point_cloud2.hpp"
+
+#include "rclcpp/rclcpp.hpp"
+#include "nav2_util/lifecycle_node.hpp"
+
+namespace nav2_map_server
+{
+
+/**
+ * @class nav2_map_server::MapServer3D
+ * @brief Parses the map yaml file and creates a service and a publisher that
+ * provides PointCloud maps, via. GetMap3D, and LoadMap3D services
```

Comment:
This is not only so: during MapServer `on_activate()` stage it also publishes loaded map once, so we can get PCL also through a running map server once. I better to change it to:
```
provides PointCloud maps, hosts GetMap3D and LoadMap3D services.
```